### PR TITLE
feat(ChangeLogs): Fixed issue related to null to empty string or collection conversion

### PR DESCRIPTION
**Pull request by JaideepPalit**
> Fixed issue related to null to empty string or collection conversion
> * Which issue is this pull request belonging to and how is it solving it? (*#1053*)
> * Did you add or update any new dependencies that are required for your change?

### Checklist
Must:
- [X] All related issues are referenced in commit messages and in PR

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>

### CODE REVIEW

**Review by mcjaeger — APPROVED** [review]

**Comment by mcjaeger** [comment]
(code has some formatting issues)

**Review by mcjaeger — CHANGES_REQUESTED** [review]
please a. not three quals, b. spaces

**Comment by mcjaeger** [comment]
(from discussion) pro and cons about having the changelog here and there, maybe, if the changelog is delivered over the REST API, maybe it is better to move it into the backend level

**Comment by JaideepPalit** [comment]
Made changes as suggested.
Moved code to backend.

**Comment by mcjaeger** [comment]
re-test looks good

File: backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ChangeLogsDatabaseHandler.java [hunk 1]
```diff
@@ -33,8 +37,13 @@
...
     private final ChangeLogsRepository changeLogsRepository;
-
-    private static final Logger log = LogManager.getLogger(ChangeLogsDatabaseHandler.class);
+    private static final ImmutableSet<String> setOfIgnoredFieldValues = ImmutableSet.<String>builder()
+            .add("\"\"")
+            .add("[]")
+            .add("{}").build();
+    private static final ImmutableSet<String> setOfIgnoredFieldNames = ImmutableSet.<String>builder()
+            .add("revision")
+            .add("documentState").build();
```
Comment (mcjaeger) [inline backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ChangeLogsDatabaseHandler.java:46]: maybe ìssetBitfield`might com ein future or whatever its name is

**Review by mcjaeger — APPROVED** [review]
code looks OK to me (at least)

**Pull request merged**